common/sai_npu.py

```diff
@@ -336,7 +336,13 @@ def remove_vlan_member(self, vlan_oid, bp_oid):
 
         assert False
 
-    def create_route(self, dest, vrf_oid, attrs=None):
+    def create_route(self, dest, vrf_oid, nh_oid=None, opt_attr=None):
```

Comment:
In such case `topologies/dc_t1.py` must be updated.
Also, please check whether `create_route()` is used in other places.

Reply:
Thanks for the comment!
create_route () occurs only in the test_dc_t1.py and dc_t1.py files. Fixed these files according to changes in the function. Test run test_dc_t1.py was successful.